src/Pages/AddWork/work.css

```diff
@@ -0,0 +1,94 @@
+.work-page {
```

Comment:
Suggestion:
maybe rather "Log Work Page"?

Reply:
I will rename it